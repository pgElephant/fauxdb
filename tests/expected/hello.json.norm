{
  "isWritablePrimary": true,
  "ismaster": true,
  "maxBsonObjectSize": 16777216,
  "ok": 1
}
