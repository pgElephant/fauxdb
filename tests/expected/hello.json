{
  "ok": 1,
  "isWritablePrimary": true,
  "ismaster": true,
  "maxBsonObjectSize": 16777216
}
